app/src/test/java/com/husseinelfeky/moviesexplorer/master/MasterViewModelTest.kt

```diff
@@ -67,7 +68,8 @@ class MainViewModelTest : AutoCloseKoinTest() {
         val result = viewModel.getAllMovies().getOrAwaitValue()
 
         // THEN - Verify that movies exist and are fetched successfully.
-        assertThat(result.isNullOrEmpty(), `is`(false))
+        assertThat(result, `is`(notNullValue()))
+        assertThat(result.size, `is`(0))
```

Comment:
This would cause the test to fail, but we can change it to the following, since the result can never be null.

```suggestion
        assertThat(result.isEmpty(), `is`(false))
```